src/cdk/a11y/interactivity-checker/interactivity-checker.spec.ts

```diff
@@ -5,6 +5,7 @@ import {InteractivityChecker} from './interactivity-checker';
 describe('InteractivityChecker', () => {
   let testContainerElement: HTMLElement;
   let checker: InteractivityChecker;
+  // TODO: refactor this to be injected
```

Comment:
This comment needs to explain _why_

https://github.com/angular/material2/blob/master/CODING_STANDARDS.md#write-useful-comments

Reply:
Good catch, will add